lang/src/org/partiql/lang/eval/NaturalExprValueComparators.kt

```diff
@@ -294,4 +311,22 @@ enum class NaturalExprValueComparators(private val nullOrder: NullOrder) : Compa
 
         throw IllegalStateException("Could not compare: $left and $right")
     }
+
+    // can think of `DESC` as the converse/result of `ASC`
```

Comment:
Was a typo from my .gist:

```suggestion
    // can think of `DESC` as the converse/reverse of `ASC`
```